CONTIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+   1. Fork the repo, develop and test your code changes.
+   2. Ensure commit messages clearly define the changes that have been made.
+   3. Create a pull request.
+   
+## Adding Features
+
+Any features added must be fully tested and documented, with examples supplied in the pull request.
+The feature must support the lowest Java version that the SDK supports (see [here](#jdk-versions)).  The feature
```

Comment:
Do you think it would be better to link to the [Travis file](https://github.com/getyoti/yoti-java-sdk/blob/master/.travis.yml) when talking about the supported version? Just thinking we might forget to update the versions here

Reply:
Fixed in https://github.com/getyoti/yoti-java-sdk/pull/144/commits/002e108fc33e5079d07eca4c653892c1b2f46731